workers/social/lib/social/models/gitprovider/constants.coffee

```diff
@@ -2,5 +2,5 @@ module.exports =
   GITHUB_HOST     : 'github.com'
   RAW_GITHUB_HOST : 'raw.githubusercontent.com'
   GITLAB_HOST     : 'gitlab.com'
-  TEMPLATE_PATH   : 'Koding.yaml'
-  README_PATH     : 'Koding.README'
+  TEMPLATE_PATH   : '.koding.yml'
```

Comment:
can we also support `yaml` sometime in the future 🙏 

( i hate when CI services fail to accept yaml )